### ISSUE TITLE
Not able to claim or sell

### ISSUE THREAD

**Issue by radiodrag** [SpatiumPrinceps/LandLord#11]
This error for the claim command
```
[23:12:06] [Server thread/INFO]: NekoSenpaiii issued server command: /ll claim
[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] Exception in command: claim 
[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] java.lang.reflect.InvocationTargetException

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at java.lang.reflect.Method.invoke(Unknown Source)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at co.aikar.commands.RegisteredCommand.invoke(RegisteredCommand.java:142)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at co.aikar.commands.BaseCommand.executeCommand(BaseCommand.java:405)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at co.aikar.commands.BaseCommand.execute(BaseCommand.java:321)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at co.aikar.commands.BukkitRootCommand.execute(BukkitRootCommand.java:78)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at co.aikar.commands.BukkitRootCommand.execute(BukkitRootCommand.java:64)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.dispatchCommand(CraftServer.java:644)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at net.minecraft.server.v1_12_R1.PlayerConnection.handleCommand(PlayerConnection.java:1385)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at net.minecraft.server.v1_12_R1.PlayerConnection.a(PlayerConnection.java:1220)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(PacketPlayInChat.java:45)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(PacketPlayInChat.java:1)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at net.minecraft.server.v1_12_R1.PlayerConnectionUtils$1.run(SourceFile:13)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at java.util.concurrent.FutureTask.run(Unknown Source)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at net.minecraft.server.v1_12_R1.SystemUtils.a(SourceFile:46)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:747)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:405)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:678)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at java.lang.Thread.run(Unknown Source)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] Caused by: java.lang.IndexOutOfBoundsException: No group 5

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at java.util.regex.Matcher.start(Unknown Source)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at java.util.regex.Matcher.appendReplacement(Unknown Source)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at java.util.regex.Matcher.replaceAll(Unknown Source)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at java.lang.String.replaceAll(Unknown Source)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at biz.princeps.landlord.commands.Claim.onClaim(Claim.java:32)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	at biz.princeps.landlord.commands.Landlordbase.onClaim(Landlordbase.java:59)

[23:12:06] [Server thread/ERROR]: [Landlord] [ACF] 	... 24 more
```
And this one for the sell command 
```
[23:14:11] [Server thread/INFO]: NekoSenpaiii issued server command: /ll sell
[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] Exception in command: ll unclaim 
[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] java.lang.reflect.InvocationTargetException

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at java.lang.reflect.Method.invoke(Unknown Source)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at co.aikar.commands.RegisteredCommand.invoke(RegisteredCommand.java:142)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at co.aikar.commands.BaseCommand.executeCommand(BaseCommand.java:405)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at co.aikar.commands.BaseCommand.execute(BaseCommand.java:321)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at co.aikar.commands.BukkitRootCommand.execute(BukkitRootCommand.java:78)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at co.aikar.commands.BukkitRootCommand.execute(BukkitRootCommand.java:64)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.dispatchCommand(CraftServer.java:644)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at net.minecraft.server.v1_12_R1.PlayerConnection.handleCommand(PlayerConnection.java:1385)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at net.minecraft.server.v1_12_R1.PlayerConnection.a(PlayerConnection.java:1220)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(PacketPlayInChat.java:45)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(PacketPlayInChat.java:1)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at net.minecraft.server.v1_12_R1.PlayerConnectionUtils$1.run(SourceFile:13)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at java.util.concurrent.FutureTask.run(Unknown Source)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at net.minecraft.server.v1_12_R1.SystemUtils.a(SourceFile:46)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:747)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:405)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:678)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at java.lang.Thread.run(Unknown Source)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] Caused by: java.lang.IndexOutOfBoundsException: No group 5

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at java.util.regex.Matcher.start(Unknown Source)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at java.util.regex.Matcher.appendReplacement(Unknown Source)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at java.util.regex.Matcher.replaceAll(Unknown Source)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at java.lang.String.replaceAll(Unknown Source)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at biz.princeps.landlord.commands.Unclaim.onUnclaim(Unclaim.java:46)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	at biz.princeps.landlord.commands.Landlordbase.onUnClaim(Landlordbase.java:74)

[23:14:12] [Server thread/ERROR]: [Landlord] [ACF] 	... 24 more
```

**Comment by SpatiumPrinceps** [comment]
Some guy on spigot has the same issue. However, everything works fine for me. 

Citation: 

That is weird. I know for sure it is working correctly, because I tested it myself. There must be something with worldguard configuration which  I / we have not figured out yet.

What is your message String on Commands.Claim.moneyTook? Did you change it? This error could only happen if you use something like a $ symbol, which is interpreted by the JVM in a different way. You dont have to use any $, because Vault will do that for you :) Hopefully this is resolved now.

http://imgur.com/a/ORBBR
ignore the hacked client, I needed it because I helped someone with a ore generation plugin xD
